export function handleError(errorMessage) {
  throw new Error(errorMessage);
}
